spec/forms/idv/in_person/adress_form_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+describe Idv::InPerson::AddressForm do
+  let(:pii) { Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS }
+  let(:good_params) do
+    {
+      address1: Faker::Address.street_address,
+      address2: Faker::Address.secondary_address,
+      zipcode: Faker::Address.zip_code,
+      state: Faker::Address.state_abbr,
+      city: Faker::Address.city,
+    }
+  end
+  let(:invalid_char) { '$' }
+  let(:bad_params) do
+    {
+      address1: invalid_char + Faker::Address.street_address,
+      address2: invalid_char + Faker::Address.secondary_address + invalid_char,
+      zipcode: Faker::Address.zip_code,
+      state: Faker::Address.state_abbr,
+      city: invalid_char + Faker::Address.city,
+    }
+  end
```

Comment:
It would be good to wrap these in an additional `context` indicating that these values and the following tests pertain specifically to transliteration. i.e. especially since the usage of `good`/`bad` here could otherwise seem ambiguous.